README.md

```diff
@@ -1,21 +1,17 @@
 
 ⚠️ $\textcolor{red}{\text{Experimental}}$ ⚠️ : Not for use in any critical, production, or otherwise important deployments
 
-# Workflow Status
-[![Build Status](https://github.com/aws-solutions/solution-spark-on-aws/workflows/Build-and-test/badge.svg)](https://github.com/aws-solutions/solution-spark-on-aws/workflows/Build-and-test/badge.svg)
-[![Merge-develop-to-stage](https://github.com/aws-solutions/solution-spark-on-aws/workflows/Merge-develop-to-stage/badge.svg)](https://github.com/aws-solutions/solution-spark-on-aws/workflows/Merge-develop-to-stage/badge.svg)
-[![Publish-and-merge-to-develop](https://github.com/aws-solutions/solution-spark-on-aws/workflows/Publish-and-merge-to-develop/badge.svg)](https://github.com/aws-solutions/solution-spark-on-aws/workflows/Publish-and-merge-to-develop/badge.svg)
-[![Cfn-nag-scan](https://github.com/aws-solutions/solution-spark-on-aws/workflows/Cfn-nag-scan/badge.svg)](https://github.com/aws-solutions/solution-spark-on-aws/workflows/Cfn-nag-scan/badge.svg)
-[![Secret-scan](https://github.com/aws-solutions/solution-spark-on-aws/workflows/Secret-scan/badge.svg)](https://github.com/aws-solutions/solution-spark-on-aws/workflows/Secret-scan/badge.svg)
-[![Lint-PR](https://github.com/aws-solutions/solution-spark-on-aws/workflows/Lint-pr/badge.svg)](https://github.com/aws-solutions/solution-spark-on-aws/workflows/Lint-pr/badge.svg)
-[![semantic-release: angular](https://img.shields.io/badge/semantic--release-angular-e10079?logo=semantic-release)](https://github.com/semantic-release/semantic-release)
-[![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
+# Codecov
+## `main branch coverage`
+[![codecov](https://codecov.io/github/aws-solutions/solution-spark-on-aws/branch/main/graph/badge.svg?flag=root)](https://app.codecov.io/github/aws-solutions/solution-spark-on-aws/tree/main)
```

Comment:
```suggestion
[![main-codecov](https://codecov.io/github/aws-solutions/solution-spark-on-aws/branch/main/graph/badge.svg?flag=root)](https://app.codecov.io/github/aws-solutions/solution-spark-on-aws/tree/main)
```
could we do something like this so we don't need add that label on top?

Reply:
There is no option to edit the codecov badge text as they are provided by codecov, that's why I had to add a label at the top